tools/offsets-tool-py/offsets-tool.py

```diff
@@ -50,18 +63,45 @@ def parse_args(self):
 			print ("File '" + args.target_path + "/config.h' doesn't exist.", file=sys.stderr)
 			sys.exit (1)
 			
-		if not args.abi in self.supported_abis:
-			print ("ABI '" + args.abi + "' is not supported.", file=sys.stderr)
-			sys.exit (1)
-
 		self.sys_includes=[]
 		self.target = None
+		self.target_args = []
+
+		sysroot_ios = "/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk"
+		sysroot_watchos = "/Platforms/WatchOS.platform/Developer/SDKs/WatchOS.sdk"
```

Comment:
should we pass in the xcode + sysroot paths from https://github.com/mono/mono/blob/master/sdks/builds/ios.mk#L144-L147 instead of duplicating it here? e.g. if you want to build with a different xcode/sdk